bin/sensei.js

```diff
@@ -229,6 +261,12 @@ cli(
         "Forwarded to Reveal.js. See https://revealjs.com/presentation-size/.",
       default: 800,
     })
+    .option("analyzeBundle", {
```

Comment:
Should be removed, now that it's using env var, no?

Reply:
Yes, I thought I had done so.